site/zenodo_rdm/previewer/mirador.py

```diff
@@ -22,12 +31,16 @@ def can_preview(file):
     # supported_extensions list needs . prefixed -
     preview_extensions = current_app.config["MIRADOR_PREVIEW_EXTENSIONS"]
     supported_extensions = ["." + ext for ext in preview_extensions]
+
+    if is_pdf_previewable(file):
+        return True
```

Comment:
question: why?

Reply:
as discusssed in IRL
see below, 
```
# Find a suitable previewer
    fileobj = PreviewFile(file_metadata, pid_value, record, url)
    for plugin in current_previewer.iter_previewers(
        previewers=[file_previewer] if file_previewer else None
    ):
        if plugin.can_preview(fileobj):
            return plugin.preview(fileobj)
```

https://github.com/inveniosoftware/invenio-app-rdm/blob/3ea347e913befc0d813bc1df6f99590731ff19b8/invenio_app_rdm/records_ui/views/records.py#L302